### DISCUSSION TITLE
Carte hub geotrek

### DISCUSSION THREAD

**Discussion by juggler31** [makinacorpus/Geotrek-website#62] [Ideas]
Aujourd'hui, geotrek n'est pas très connu du grand public, il est compliqué de différencier l'outil des instances en place.

L'idée serait de créer une application et / un site dédié qui afficherait une carte avec le lien vers tout les geotrek et qui servirait de point d'entrée pour un utilisateur final afin de trouver sa rando idéale. Pour ceux qui font le lien, il s'agit d'une amélioration de la carte des utilisateurs qui soit plus intuitive et fonctionnelle !

Ce serait évidemment pour que les utilisateurs finaux arrivent sur les portails rando de chaque geotrek (PAS un geotrek global a tous)

**Comment by camillemonchicourt** [comment]
Selon moi, il ne faut pas faire autre chose que la carte présente sur https://geotrek.fr/utilisateurs.html
Par contre, il faudrait certainement l'améliorer et peut-être la fournit sous forme de liste des utilisateurs en complément.

On avait eu pas mal de discussions sur le sujet, car la gestion des données de cette carte est dans un outil à accès limité, donc pas ouvert à tous.

J'avais proposé qu'on se base plutôt sur un fichier GeoJSON commun, accessible dans ce Github, que chacun pourrait enrichir.

Sachant que je gère déjà par ailleurs un tableau et une couche SIG des utilisateurs de Geotrek pour mettre à jour la carte des utilisateurs présentée à chaque rencontre Geotrek : https://geotrek.ecrins-parcnational.fr/rencontres/2021/Geotrek-utilisateurs-2021.png

L'outil de gestion des utilisateurs actuel me semble trop limité, restreint et complexe pour être l'outil central et communautaire de gestion de la liste/carte des utilisateurs.

C'est pour ça que j'ai commencé un WIKI listant les nouveaux Geotrek que je croise, que @babastienne a complété : https://github.com/GeotrekCE/Geotrek-website/wiki/Liste-des-Geotrek-connus

Reste à faire un fichier partagé des utilisateurs, bien le consolider avec les différentes listes dont on dispose, le partager pour ensuite l'enrichir facilement et régulièrement.

Et faire en sorte que la carte des utilisateurs sur geotrek.fr s'appuie dessus. 😀
